package/MDAnalysis/coordinates/PDB.py

```diff
@@ -832,10 +833,16 @@ def _write_pdb_header(self):
         if not self.obj or not hasattr(self.obj, 'universe'):
             self._write_pdb_title(self)
             return
+        if self.first_frame_done == False:
+            self.first_frame_done = True
+        else:
+          return
 
         u = self.obj.universe
         self.HEADER(u.trajectory)
-        self._write_pdb_title()
+        if self.first_frame_done == False:
```

Comment:
this check isn't necessary anymore. Due to the check about this code is unreachable.